plugin-tester-scala/src/main/scala/example/myapp/helloworld/LiftedGreeterClient.scala

```diff
@@ -0,0 +1,99 @@
+package example.myapp.helloworld
+
+import java.util.concurrent.TimeUnit
+
+import scala.concurrent.Await
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.util.Try
+import scala.util.control.NonFatal
+
+import akka.Done
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.grpc.GrpcClientSettings
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.Source
+import io.grpc.CallOptions
+import io.grpc.StatusRuntimeException
+import io.grpc.netty.shaded.io.grpc.netty.GrpcSslContexts
+import io.grpc.netty.shaded.io.grpc.netty.NegotiationType
+import io.grpc.netty.shaded.io.grpc.netty.NettyChannelBuilder
+import io.grpc.netty.shaded.io.netty.handler.ssl.SslContext
+
+import example.myapp.helloworld.grpc._
+
+object LiftedGreeterClient {
+
+  def main(args: Array[String]): Unit = {
+
+    implicit val sys = ActorSystem()
+    implicit val mat = ActorMaterializer()
+    implicit val ec = sys.dispatcher
+
+    val client = new GreeterServiceClient(GrpcClientSettings(
+      "127.0.0.1",
+      8080,
+      overrideAuthority = Some("foo.test.google.fr"),
+      None,
```

Comment:
perhaps name this parameter as well?

Reply:
ups, keep missing out on the plugin-testers as they are not part of the aggregated build